data/assets/scene/solarsystem/planets/jupiter/jupiter.asset

```diff
@@ -48,10 +48,14 @@ local JupiterLabel = {
     Type = "RenderableLabel",
     Text = "Jupiter",
     FontSize = 70.0,
-    Size = 8.77,
-    MinMaxSize = { 1, 100 },
+    Size = 8.75,
+    MinMaxSize = { 1, 50},
```

Comment:
I like to have a space between the number and the opening and closing `{`/`}`

Reply:
Fixed in: [055eab06540015eba3138c73cb4cb4bf6f36b5ad](https://github.com/OpenSpace/OpenSpace/commit/055eab06540015eba3138c73cb4cb4bf6f36b5ad)